Function/Program.cs

```diff
@@ -2,14 +2,22 @@
 using Microsoft.Extensions.Hosting;
 using Microsoft.Extensions.DependencyInjection;
 using Microsoft.Azure.Functions.Worker.Extensions.OpenApi.Extensions;
+using Microsoft.Extensions.Azure;
+using Microsoft.Extensions.Configuration;
 
 // This defines the host that runs in the azure function host process
 var host = new HostBuilder()
     .ConfigureFunctionsWorkerDefaults(worker => worker.UseNewtonsoftJson())
     // services for dependecy injection
-    .ConfigureServices(services => {
+    .ConfigureServices((hostContext, services) =>
+    {
         services.AddApplicationInsightsTelemetryWorkerService();
         services.ConfigureFunctionsApplicationInsights();
+        services.AddAzureClients(clientBuilder =>
+        {
+            clientBuilder.AddTableServiceClient(hostContext.Configuration
+            .GetWebJobsConnectionString("AzureWebJobsStorage"));
+        });

```

Comment:
Here I am adding the table storage client to the dotnet worker. The authentication against the service is done with a connection string read from `AzureWebJobsStorage`. In development that string is set to `UseDevelopmentStorage=true` so Azurite is used and in production that string is set in the function configuration in the azure portal to the storage account in your resource group.